ui/changelog.md

```diff
@@ -1,4 +1,13 @@
 # UI Colors Changelog
 
+### 3.0.0
+
+Added new color definition for vantaBlack color palette for dark themes.
+
+** Breaking change: **
+
+-   Removed the oledBlack and all the accent colors.
```

Comment:
```suggestion
-   Replaced oledBlack palette with darkBlack
-   Accent colors have been removed
```

Reply:
Or "Replaced oledBlack palette with vantaBlack"?

Reply:
I think vantaBlack is too fun (and I hate to be the squasher of fun). I think darkBlack is still the more appropriate name to go with the rest of the UI palette colors.